dev/tasks/tasks.yml

```diff
@@ -888,12 +888,12 @@ tasks:
       - r-lib__libarrow__bin__darwin-arm64-openssl-3.0__arrow-{no_rc_r_version}\.zip
       - r-lib__libarrow__bin__darwin-x86_64-openssl-1.1__arrow-{no_rc_r_version}\.zip
       - r-lib__libarrow__bin__darwin-x86_64-openssl-3.0__arrow-{no_rc_r_version}\.zip
+      - r-pkg__bin__windows__contrib__4.4__arrow_{no_rc_r_version}\.zip
       - r-pkg__bin__windows__contrib__4.3__arrow_{no_rc_r_version}\.zip
-      - r-pkg__bin__windows__contrib__4.2__arrow_{no_rc_r_version}\.zip
-      - r-pkg__bin__macosx__big-sur-x86_64__contrib__4.3__arrow_{no_rc_r_version}\.tgz
-      - r-pkg__bin__macosx__contrib__4.2__arrow_{no_rc_r_version}\.tgz
+      - r-pkg__bin__macosx__big-sur-x86_64__contrib__4.4__arrow_{no_rc_r_version}\.tgz
+      - r-pkg__bin__macosx__contrib__4.3__arrow_{no_rc_r_version}\.tgz
+      - r-pkg__bin__macosx__big-sur-arm64__contrib__4.4__arrow_{no_rc_r_version}\.tgz
```

Comment:
I don't think we've had a discussion about if we should _also_ keep binaries for older versions of R. I know CRAN only distributes current and old, but we say we support the previous four versions — should we also host binaries for those too? 

In this case it's "just" not removing 4.2 here. Not sure if it's worth going and adding 4.1 and 4.0 too, but at this point we know that 4.2 works — do we actually want to remove it?

cc @assignUser

Reply:
The `r-pkg-*` package binaries are just used for the nightlies where we just keep rel and old-rel so dropping 4.2 is fine. `r-lib-*` are the ones we use for the release etc. and are not bound to a specific R version.